src/locales/en-US/list.ftl

```diff
@@ -2,10 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-## common
+## Firefox / Lockbox Brand
+##
+## Firefox and Lockbox must be treated as a brand, and kept in English.
+## They cannot be:
+## - Declined to adapt to grammatical case.
+## - Transliterated.
+## - Translated.
+##
+## Reference: https://www.mozilla.org/styleguide/communications/translation/
+-brand-name = Firefox
+-product-short-name = Lockbox
+-product-full-name = Firefox Lockbox
+# TODO: should Firefox Account also not be translated or declined?
+-product-accounts = Firefox Account
+
+header-logins-button = Logins
```

Comment:
Add another section comment, saying that these strings are localizable? Otherwise, the previous comment apply to these strings too.

The alternative is to move unlocalizable brands at the end of the file, or in a separate file.